src/Mvc/Mvc.Core/src/ModelBinding/FormModelBindingHelper.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding;
+
+internal static class FormModelBindingHelper
```

Comment:
Suggest choosing a more general name since this is also used in HTML generation

Reply:
I've reorganized things a bit so the decision making around what should use "invariant" formatting doesn't occur in multiple places in the code (it now all exists in `DefaultHtmlGenerator`, and its decision is cached in `FormContext` so the `InputTagHelper` knows to generate the extra hidden element). As such, I've removed this type and placed the `CultureInvariantFieldName` string in `FormValueProvider`. Does that sound fine?